### ISSUE TITLE
Document how to disable cmor checks and fixes

### ISSUE THREAD

**Issue by bouweandela** [ESMValGroup/ESMValCore#88]
For scientists who like living on the edge

**Comment by bouweandela** [comment]
To disable the CMOR checks, use the following preprocessor:
```
fix_metadata: {cmor_table: null}
fix_data: {cmor_table: null}
cmor_check_metadata: false
cmor_check_data: false
```
the cmor_table argument to `fix_(meta)data` is needed because part of the automatic fixes is done by the cmor checkers (don't ask me why).

To disable both fixes and CMOR checks:
```
fix_file: false
fix_metadata: false
fix_data: false
cmor_check_metadata: false
cmor_check_data: false
```

**Comment by valeriupredoi** [comment]
superseded and implemented via cmor check levels by #374 - @bouweandela we can close this
